optaplanner-website-root/content/competitor/gurobi.adoc

```diff
@@ -341,7 +341,7 @@ OptaPlanner supports any number of score levels:
 * n levels with `BendableScore`
 
 This allows users to prioritize operational constraints (such as _assign all shifts_)
-over financial constraints (such as _reduce cost_), without multiplication to with a big number.
+over financial constraints (such as _reduce cost_), without multiplication with a big number.
```

Comment:
Not sure what was the intention. "Multiplication with a big number" makes a bit more sense to me. Let me know if you want me to change that to "to a big number".

Reply:
This fix is correct, thanks :)